README.EN.md

```diff
@@ -0,0 +1,549 @@
+
+Django is one of the most re web frameworks available on the market. It manages everything from the database to the final HTML sent to the client. But with the introduction of the concept of [*single page apps*](wikipedia) to the front-end development environment, it has become usual to find systems using Django only to provide an [API](wikipedia) on the back-end that responds JSON data to be consumed by Javascript applications. This architecture that divides front and back-end into two separate code bases enables the independence between these sectors on the development process. It also makes it possible to have multiple client apps interacting with one API, ensuring the consistency of data and business rules and at the same time providing different user interfaces.
+
+Having two code bases though creates some extra work. There's one more environment to configure and another deploy process to care about. A way of making this simpler is to use the resources provided by Django for serving static files. After all, a front-end application is just a set of those files.
+
+The goal of this post is showing step-by-step how to create a basic setup for a [Django project](https://www.djangoproject.com/) including all the necessary resources for a [React app](https://reactjs.org/docs/hello-world.html) managed by [Webpack](https://webpack.js.org/). I'll show how to configure Django to serve both the API and the front-end files.
+
+For those who are more used to the Python  environment, the nest of Javascript libraries can be a big challenge to grasp. There are many ways of combining the same resources and get similar results. And the references we encounter on the web are frequently obscure on the purpose of each part of the setup. That's what I'm going to try to show in this post.
+
+The first part of this post is intended to show how to do the necessary setup for Django to provide the front-end application through its resources for serving static files. After that, we're going to build the basic structure of a React application focusing on showing the roles of each element in it. You won't find here a detailed explanation of how React works, only some core concepts to understand the configuration. The purpose of this post is to empower someone with more background on the back-end to be able to integrate with the front-end more easily without getting lost in the development environment setup. From that point, you can search for other tutorials more specific to the framework itself or even read the documentation, which is very detailed and understandable.
+
+## Django and static files
+
+On a project architecture using Django's built-in [*template engine*](https://docs.djangoproject.com/en/dev/ref/templates/) the static files used on the templates, like `.css` and images, are stored in a directory specified on the `STATICFILES_DIRS` settings variable. You only have to define that variable and Django will serve the files for you.
```

Comment:
The staticfiles can be also stored into apps static folders, and those folders they doesn't need to be specified on the STATICFILES_DIRS variable.